### ISSUE TITLE
Playlist does not continue playing after a video ends

### ISSUE THREAD

**Issue by fabianski7** [smplayer-dev/smplayer#426]
**To Reproduce**
1. Enable Autoplay next episode
2. Open a playlist
3. Play a video and go to the end and wait for it to finish

**Screenshots**
https://upload.vaa.red/GgXRA#iSaCRWhDKWcbWKYchafHHTRZ45RtYZbh

**Your Enviroment**
 - SMPlayer version: tested [with 22.2.0](https://archlinux.org/packages/community/x86_64/smplayer/) and [git version](https://aur.archlinux.org/packages/smplayer-svn)
 - OS: Archlinux
 - Multimedia engine:
 [ ] MPlayer
 [x] MPV

**Comment by fabianski7** [comment]
I managed to solve this by passing the `--no-config` option to the MPV, so smplayer might be loading my user MPV config file, I thought it didn't.

what do you think @smplayer-dev?

**Comment by smplayer-dev** [comment]
Since the latest version, the `--no-config` option isn't passed to mpv anymore, so yes, the mpv config is used.

**Comment by fabianski7** [comment]
I think this might be a problem for linux users, as it is quite common for them to modify this file

**Comment by smplayer-dev** [comment]
I agree, but several people complained that the mpv config wasn't used so I made that change. But maybe I'll revert it since it causes more problems than it solves.
